crates/dkg-cli/src/user/actions.rs

```diff
@@ -1,20 +1,22 @@
 use super::opts::{FinalizeOpts, NewOpts, PublishSharesOpts, StateOpts};
-use crate::CLIResult;
 use rand::RngCore;
 use std::{fs::File, io::Write};
 
 use dkg_core::{
-    node::{DKGPhase, Phase0, Phase1, Phase2, Phase2Result, Phase3},
+    dkg_impls::joint_feldman::{DKGWaitingJustification, DKGWaitingResponse, DKGWaitingShare, DKG},
```

Comment:
I have to say I dont like that at all - yet another import. DKG lib is already separated into two folders and now there is a third one... We're very far from simplicity now.

Reply:
wdyt about https://github.com/celo-org/celo-threshold-bls-rs/pull/79/commits/146735bfb3fc4b830842c0f1a7b65ff36a398ac7?